Hello day 2 at Microverse

XRichardroidX opened the pull request:
In this Microverse Task for today I added,
1. an HTML file which displays the text, "Hello Microverse!".
2. a CSS file which makes the color of the Text blue.
   And all of these were done using the help of Linters for code uniformity and accuracy.

XRichardroidX committed:
I added HTML and CSS files

XRichardroidX committed:
Update index.html

jerryowusu commented on index.html:
```diff
@@ -0,0 +1,12 @@
+<body>
+    <h1>Hello Microverse</h1>
+</body>
+</html>
```
[OPTIONAL] 
![Screenshot from 2022-09-20 17-35-51](https://user-images.githubusercontent.com/65233860/191326254-6f75a6cc-161a-4a66-b90a-897a0337fffd.png)

As shown above, the blue color you added to the h1 is not working since you have not linked the stylesheet to the html file. Kindly fix that

jerryowusu reviewed (changes requested):
Hi @XRichardroidX , 

Good job so far!
I really like how neat and organised your code is. Keep it up! However, there are some few issues that you still need to work on to go to the next project but you are almost there!

![so close](https://user-images.githubusercontent.com/65233860/191327684-069ed7aa-6707-4716-b802-ca810e2035ed.gif)


## Highlights

- All linters are passing :heavy_check_mark: 
- Good GitHub flow :heavy_check_mark: 

## Required Changes ♻️

- [ ] ![Screenshot from 2022-09-20 17-40-18](https://user-images.githubusercontent.com/65233860/191328745-b7067f6f-4866-4305-8fa4-f35a29f7b6b8.png)
Kindly provide a description for your pull request. You should always ensure that your pull request has a detailed description of what changes are inside.

- [ ] Your readme file is not complete. Kindly make sure that you have edited the file by providing a detailed description of the project, what tech stacks was used, how to start and set up etc. The template given you is just a sample that you have to edit for each project you make. You can consider this [link](https://github.com/jerryowusu/personal-portfolio/edit/main/README.md) as an example to edit yours


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

XRichardroidX committed:
I added more information to the read me file

I have made changes to this ReadMe file by adding more information including a link to my github, Linkedin and twitter account.

XRichardroidX committed:
I added the style link to make the text blue

I forgot to add the css link to make the html text blue, but now that has been added, enjoy your blue text.

GiuseppeTG reviewed (approved):
Hi @XRichardroidX  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥇 : 
Congratulations! 🎉 

![Alt Text](https://media3.giphy.com/media/nXxOjZrbnbRxS/giphy.gif)

## Highlights
✅ Good job implementing the changes required
✅ Professional PR and readme
✅ No linter errors

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

XRichardroidX commented:
@GiuseppeTG 
  Do I need to delete the Feature-Branch after merging it to the main branch?
